Code/RDBoost/python_streambuf.h

```diff
@@ -195,20 +195,21 @@ class streambuf : public std::basic_streambuf<char> {
       : streambuf(python_file_obj, buffer_size_) {
 #if 1
     bp::object io_mod = bp::import("io");
-    CHECK_INVARIANT(io_mod,"module not found");
-    bp::object iobase = io_mod.attr("TextIOBase");;
-    CHECK_INVARIANT(iobase,"base class not found");
-#else 
+    CHECK_INVARIANT(io_mod, "module not found");
+    bp::object iobase = io_mod.attr("TextIOBase");
+    ;
```

Comment:
???

Reply:
That was clang-format: it turned the extra semicolon into a separate line. I'll remove it.